spec/unit/module_tool/tar/gnu_spec.rb

```diff
@@ -1,18 +1,19 @@
 require 'spec_helper'
 require 'puppet/module_tool'
 
-describe Puppet::ModuleTool::Tar::Gnu do
+describe Puppet::ModuleTool::Tar::Gnu, unless: Puppet::Util::Platform.windows? do
```

Comment:
Why would this matter for unit tests? Don't you want to run the unit tests on Windows too?

Reply:
In general I agree we should test unit tests everywhere, but that would mean stubbing out commands that don't exist on Windows, so seems kind of pointless. A common problems is that test has to account for `File.expand_path` taking into account the current drive letter, which is `D:` in GH runners.